examples/typescript-backbone/js/app.ts

```diff
@@ -339,6 +396,20 @@ class AppView extends Backbone.View {
 		Todos.each(this.addOne);
 	}
 
+	// Filter out completed/remaining tasks
+	filter(criteria: string) {
+		taskFilter = criteria;
```

Comment:
At least when people ask me which frameworks they should use I can now give them a list that I think they shouldn't use. Learned a lot of that lately. :smiling_imp: